src/routes/main.ts

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const router = express.Router();
+const Issuer = require('../models/issuer');
+const tools = require('../bin/tools');
+const badgeclass_controller = require('../controllers/badgeclassController');
+const assertion_controller = require('../controllers/assertionController');
+import {Request, Response} from 'express';
+
+// GET request for "home page": give links to all possibilities
+router.get('/', (req: Request, res: Response): void => {
+    res.status(200).send({ 
+      assertions: tools.server_url + '/assertions' ,
+      badgeclasses: tools.server_url + '/badgeclasses',
+      issuer: tools.server_url + '/issuer',
+    });
+  });
+
+// GET request for issuer (there will only be one for now - WISE badges)
+router.get('/issuer', (req: Request, res: Response): void => {
+  Issuer.findOne()
+  .exec(function (err: Error, issuer: Object) {
+    //TODO: is this error handling correct
+      if (err) { res.status(404) }
+      else {
+        res.json(issuer)
+      }
```

Comment:
Without sending the client will keep waiting for a response. You can also return here, making the `else` clause unnecessary.

```suggestion
      if (err) { return res.status(404).send(); }
      return res.json(issuer);
```